mutiny-core/src/nodemanager.rs

```diff
@@ -71,33 +71,26 @@ pub(crate) struct NodeIndex {
 
 // This is the NodeIdentity that refer to a specific node
 // Used for public facing identification.
-#[wasm_bindgen]
 pub struct NodeIdentity {
```

Comment:
for all these types we could change them to be more type-safe and instead of using String use the actual types

Reply:
same with all the functions, we do things like `invoice_str: String` but could take in an `invoice: Invoice`.

Reply:
Might be a lot so we could do this in a follow up

Reply:
I did change a couple `String` that was a `TxId`. A follow up to identify more of them and be type safe everyone would be nice.

Reply:
https://github.com/MutinyWallet/mutiny-node/issues/360